docs/audit-log-filter/audit-log-filter-options.md

```diff
@@ -0,0 +1,419 @@
+
+| Option name | Description |
+| --- | --- |
+| Command-line | --audit-log-filter-disable |
+| Dynamic | Yes  |
+| Scope | Global  |
+| Data type | Boolean  |
+| Default | OFF |
+
+Disables logging for the plugin for all connections and any sessions. 
+
+Using this variable requires the user account to have `SYSTEM_VARIABLES_ADMIN` and `AUDIT_ADMIN` privileges.
+
+### audit_log_filter_encryption
+
+| Option name | Description |
+| --- | --- | 
+| Command-line | --audit-log-filter-encryption |
+| Dynamic | No  |
+| Scope | Global  |
+| Data type | Enumeration  |
+| Default | NONE  |
+| Valid values | NONE or AES |
+
+Defines the encryption type for the audit log filter file. The values can be either `NONE`, the default value and no encryption, or `AES`.
+
+### audit_log_filter_file
+
+| Option name | Description |
+| --- | --- | 
+| Command-line | --audit-log-filter-file |
+| Dynamic | Yes  |
+| Scope | Global  |
+| Data type | Enumeration  |
+| Default | ON  |
+
+Defines the name and suffix of the audit log filter file. The plugin writes events to this file. 
+
+The file name and suffix can be a relative path name, the plugin looks for this file in the data directory, or a full path name, the plugin uses the given value. If you use a full path name, be sure that the directory is accessible only to users who need to view the log and the server.
+
+For more information, see [Naming conventions](audit-log-filter-naming.md)
+
+### audit_log_filter_filter_id
+
+| Option name | Description |
+| --- | --- | 
+| Command-line | --audit-log-filter-file-id |
+| Dynamic | Yes  |
+| Scope | Global  |
+| Data type | Integer  |
+| Default | 1  |
+| Minimum value | 0 |
+| Maximum value | 4292967295 |
+
+Defines the current session's internal ID of the audit log filter file. 
+
+The default value is 0 (zero), which means that the session has no filter assigned.
+
+### audit_log_filter_format
+
+| Option name | Description |
+| --- | --- | 
+| Command-line | --audit-log-filter-format |
+| Dynamic | Yes  |
+| Scope | Global  |
+| Data type | Enumeration  |
+| Default | NEW  |
+| Available values | OLD, NEW, JSON |
+
+Defines the audit log filter file format. 
+
+The available values are: [OLD (old-style XML)](audit-log-filter-old.md), [NEW (new-style XML)](audit-log-filter-new.md) and [JSON](audit-log-filter-json.md).
+
+### audit_log_filter_format_unix_timestamp
+
+| Option name | Description |
+| --- | --- | 
+| Command-line | --audit-log-filter-format-unix-timestamp |
+| Dynamic | Yes  |
+| Scope | Global  |
+| Data type | Boolean  |
+| Default | OFF  |
+
+This option is only supported for JSON-format files.
+
+Enabling this option adds a `time` field to JSON-format files. The `time` field indicates the date and time when the event was generated. Changing the value causes a file rotation because all records must either have or do not have the `time` field. This option requires the `AUDIT_ADMIN` privilege along with the `SYSTEM_VARIABLES_ADMIN` privileges. 
```

Comment:
indicates the date and time when the event was generated - an integer that represents the UNIX timestamp value indicating the date and time when the audit event was generated

Reply:
Done.